lib/forwarder.ts

```diff
@@ -13,6 +13,9 @@ import * as crypto from "crypto";
 import { LambdaDestination } from "@aws-cdk/aws-logs-destinations";
 const SUBSCRIPTION_FILTER_PREFIX = "DatadogSubscriptionFilter";
 export function addForwarder(scope: cdk.Construct, lambdaFunctions: lambda.Function[], forwarderARN: string) {
+  if (scope.node.tryFindChild("forwarder")) {
```

Comment:
This means the rest of this function's logic (subscribing the forwarder to log groups) won't execute on the second call, do you know if the function is called with the same `lambdaFunctions` and `forwarderARN` on the subsequent calls? in other words, do you know if it's safe to NOT create the subscription filters for the subsequent calls because they are already created?